### ISSUE TITLE
[MU4 Issue] Keyboard shortcut consistency suggestion: alt+shift+arrow-key

### ISSUE THREAD

**Issue by WillyWonksters** [musescore/MuseScore#11774]
The left/right keys move from chord/note to chord/note. Shift+left/right performs the same movement but also selects those chords/notes.
Since Alt+arrow-key also moves up/down across the individual notes in a chord, shouldn’t alt+shift+arrow-key also select those individual notes?

**Platform information**
MuseScore 4.0.0 OS: Debian GNU/Linux 11 (bullseye), Arch.: x86_64, MuseScore version (64-bit): 4.0.0-2059995262, revision: github-musescore-musescore-cafefd7

**Comment by MarcSabatella** [comment]
There's something logical about this to be sure, but - the way Shift+Left/Right work, it's always about a range selection, and since each press of Left or Right by definition move the cursor to a different time position, each press of Shift+Left/Right will always change an endpoint of the range.  That wouldn't be true for Alt+Shfit+Left/Right.  You might have a range selected and then Alt+Right moves the selection to a hairpin - what would that mean?  Or, you might be moving from note to note within a single chord which doesn't change the time position, so this would have no effect on the range endpoints.

But, one thing this does bring up is the idea that making a *list* selection - normally by Ctrl+clicking individual elements that may or may not be contiguous - is difficult or impossible to do by keyboard.  That's because any attempt to navigate to another element after selecting one loses the original selection, because selection *is* navigation.  Meaning, list selections are pretty much not accessibile to blind users (except to the extent you can create them by right-click & the Select menu).

But, what if Alt+Shift+Left/Right (or some other shortcut) moved the "selection cursor" but left the existing selection alone otherwise?  So if you are current on an element, then you press Alt+Shift+Right, it would leave that element selected and then also select the next element.  From there, Alt+Right could navigate while leaving that first element selected.  Then the next Alt+Shift+Right would add the current element to the list selection and move the cursor again, etc.  So Alt+Shfit+Right would basically be the equivalent of "Ctrl+click plus select next element as the cursor", Alt+right would always move the active element but leave anything else that was previous selected.

Sounds complicated, I know, but in my imagination, it's seems relatively natural, and would be one solution to a problem I don't think anyone has really thought much about solving before.

**Comment by MarcSabatella** [comment]
Anyhow, there is definitely something to consider here, but probably it needs to be thought out and designed somewhere other than here in this issue description - maybe GitHub discussions the Development forum, Discord, etc.  With @bkunda, @Tantacrul,  and/or @shoogle ultimately making the final call probably.

**Comment by bkunda** [comment]
Good call, @MarcSabatella, and thanks @WillyWonksters for the suggestion!
I'll move this to discussions for now.
